src/ray/pubsub/publisher.h

```diff
@@ -54,31 +56,45 @@ class SubscriptionIndex {
   /// Erase the object id and subscriber id from the index. Return the number of erased
   /// entries.
   /// NOTE: It cannot erase subscribers that were never added.
-  bool EraseEntry(const ObjectID &object_id, const SubscriberID &subscriber_id);
+  bool EraseEntry(const std::string &message_id_binary,
+                  const SubscriberID &subscriber_id);
 
   /// Return true if the object id exists in the index.
-  bool HasObjectId(const ObjectID &object_id) const;
+  bool HasMessageId(const std::string &message_id_binary) const;
```

Comment:
Is this API used outside of testing? If no, could you please move it into private and mark it as a testing API?